manifests/apache/vhost.pp

```diff
@@ -88,6 +96,8 @@
   Optional[String] $ldap_bind_password      = undef,
   Optional[String] $ldap_url                = undef,
   Optional[String] $ldap_bind_authoritative = undef,
+  Boolean $ldap_require_group               = $::puppetboard::params::ldap_require_group,
```

Comment:
I believe the leading colons are not needed here anymore, so 
`$puppetboard::params...`